source/docs/articles/wordpress/solr.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Enabling Solr for WordPress
+description: Detailed information on using Apache Solr with WordPress.
+category:
+    - wordpress
+    - developing
+keywords: apche, apache solr, index, indexing, searching, index and search, indexing and searching, solr, how to enable solr, enable solr, solr api, indexserver solr, solr indexserver, solr api, enable solr search, apachesolr, apache solor search module, solr modules, configure solr,
+---
+[Apache Solr](/docs/articles/sites/apache-solr) is a system for indexing and searching site content. First, you will need to add the Index Server to your site. From your Dashboard, go to **Settings** > **Add Ons** > **Apache Solr Index Server: Add**.
+
+This will provision Apache Solr containers for every environment for your site. You are now ready to begin integrating with WordPress.
+
+## Solr Search for WordPress
+Pantheon supports and maintains [Solr Search for WordPress (Solr Power)](https://wordpress.org/plugins/solr-power/).  This plugin replaces the [default search mechanism](https://codex.wordpress.org/Class_Reference/WP_Query#Search_Parameter) within WordPress while preserving the familiar integration methods within themes and widgets.
+
+
+## Install and Configure Plugin
+1. [Set the connection mode to SFTP](/docs/articles/sites/code/developing-directly-with-sftp-mode) for the Dev or Multidev environment via the Pantheon Dashboard or with [Terminus](/docs/articles/local/cli/):
+ ```bash
+ terminus site set-connection-mode --site=<site> --env=dev --mode=sftp
+ ```
+
+2. Install and activate the [Solr Search for WordPress (Solr Power)](https://wordpress.org/plugins/solr-power/) plugin on the Dev or Multidev environment using the WordPress Dashboard or with Terminus:
+ ```bash
+ terminus wp 'plugin install --activate solr-power' --site=<site> --env=dev
+ ```
+ Upon activation, the plugin will generate and send a `schema.xml` document to the Solr server.

```

Comment:
Consider linking to the schema.xml file: https://github.com/pantheon-systems/solr-power/blob/master/schema.xml